analyses/mwi/dryspells_trigger/exploration/test_resampling.md

```diff
@@ -0,0 +1,270 @@
+```python
+%load_ext autoreload
```

Comment:
I am unsure what the diff between this file and `mwi_agg_exploration.Rmd` is. Explain a bit and fix errors, then I can review again